lib/parity/usage.rb

```diff
@@ -1,7 +1,7 @@
 module Parity
   class Usage
     def to_s
-      File.read(readme).match(/Usage\n-----\n([^\/.]*)staging open/)[1]
+      File.read(readme).match(/Usage\n-----\n([^\/]+)\n\[\S+\]/)[1]
```

Comment:
I consider myself pretty OK at reading regex, and I find this hard to read.

What about grabbing everything between `Usage\n-----\n` and `^Convention`?